letsencrypt-auto

```diff
@@ -67,9 +74,9 @@ then
 
   echo "Creating virtual environment..."
   if [ "$VERBOSE" = 1 ] ; then
-    virtualenv --no-site-packages --python python2 $VENV_PATH
+    virtualenv --no-site-packages --python ${LE_PYTHON:-python2} $VENV_PATH
```

Comment:
Set LE_PYTHON correctly in the code above, rather than messing around with it as you're using it?